src/sentry/api/endpoints/team_projects.py

```diff
@@ -218,14 +218,8 @@ def post(self, request: Request, team: Team) -> Response:
                 sender=self,
             )
 
-            # Create project option to turn on ML similarity feature for new EA projects
-            is_seer_eligible_platform = project.platform in SEER_ELIGIBLE_PLATFORMS
-            if (
-                hasattr(project.organization, "flags")
-                and project.organization.flags.early_adopter
-                and is_seer_eligible_platform
-                and options.get("similarity.new_project_seer_grouping.enabled")
-            ):
+            # Create project option to turn on ML similarity feature for new projects
+            if project.platform in SEER_ELIGIBLE_PLATFORMS:
```

Comment:
Do we still need the options check here for single-tenants?

Reply:
good point, thanks! do you happen to know which [regions](https://github.com/getsentry/sentry-options-automator/tree/10f3e82690af383fd113af8e00164e5f87356dc9/options/regions) I'd need to set the option to false for in options automator?